### PR TITLE
CB-31533: branding & terminology changes

### PR DESCRIPTION
**Pull request by dseidel-b9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,17 +44,17 @@ If you suspect a problem, please first look at the ThreatConnect connector logs
 
 ## Support
 
-1. View all API and integration offerings on the [Developer Network](https://developer.carbonblack.com/) along with reference documentation, video tutorials, and how-to guides.
-2. Use the [Developer Community Forum](https://community.carbonblack.com/t5/Developer-Relations/bd-p/developer-relations) to discuss issues and get answers from other API developers in the Carbon Black Community.
-3. Report bugs and change requests to [Carbon Black Support](http://carbonblack.com/resources/support/).
+* View all API and integration offerings on the [Developer Network](ttps://developer.carbonblack.com) along with reference documentation, video tutorials, and how-to guides.
```
Comment (sjones-github) [inline README.md]: ttps://developer.carbonblack.com should likely be https://developer.carbonblack.com

File: README.md [hunk 2]
```diff
@@ -44,17 +44,17 @@ If you suspect a problem, please first look at the ThreatConnect connector logs
...
+* Use the [Developer Community Forum](https://community.carbonblack.com/community/resources/developer-relations) to discuss issues and get answers from other API developers in the Carbon Black Community.
+* Report bugs and change requests to [Carbon Black Support](http://carbonblack.com/resources/support/).
```
Comment (sjones-github) [inline README.md:49]: http://carbonblack.com/resources/support/ redirects to https://carbonblack.com/support/

File: README.md (hunk 1 above)
Comment (dseidel-b9) [reply README.md]: Thanks, will push a fix in a minute.

File: README.md (hunk 2 above)
Comment (dseidel-b9) [reply README.md:49]: This is the link that was in the ticket.

**Review by zacharyestep — APPROVED** [review]

**Review by sjones-github — APPROVED** [review]

**Pull request merged**